.github/workflows/go.yml

```diff
@@ -0,0 +1,37 @@
+name: Go
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
+      run:  go test -v ./...
+
+    - name: Fuzz
+      run: |
+        export GOPATH=$GOROOT
+        make -f Makefile.fuzz get
+        make -f Makefile.fuzz
```

Comment:
this doesn't fuzz on builds the fuzzing parts, it this correct?

Reply:
Correct. This only checks that the build of fuzz targets is correct.

For running the fuzz targets, there is the CIfuzz part, using corpus from ossfuzz